client/components/Builder/BuilderPedals.js

```diff
@@ -32,26 +40,34 @@ const BuilderPedals = ({ mouseLeave, mouseOver, rotate, pedals }) => {
       {pedals.map((elem, index) => (
         <Draggable key={elem.id}>
           <div>
-            {elem.showRotate ? (
+            {elem.showButtons && (
               <Icon
                 onMouseEnter={event => mouseOver(elem.id, event)}
                 onMouseLeave={event => mouseLeave(elem.id, event)}
-                style={styles.icon}
+                style={styles.iconRotate}
                 onClick={event => rotate(elem.id, event)}
-                name="undo"
+                name="refresh"
               />
-            ) : null}
+            )}
             <Image
               onMouseEnter={event => mouseOver(elem.id, event)}
               onMouseLeave={event => mouseLeave(elem.id, event)}
-              id={elem.id}
               item={elem}
               src={elem.image}
               style={{
                 ...styles.image,
                 transform: `rotate(${elem.rotation}deg)`
               }}
             />
+            {elem.showButtons && (
+              <Icon
```

Comment:
Break these icons into a single separate component that you can conditionally overlay on top of the pedal.